cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -167,6 +173,26 @@ describe(["@tier1"], "Application assessment and review tests", () => {
         // bug: https://issues.redhat.com/browse/MTA-1751
     });
 
+    // Polarion TC MTA-392
+    it("View archived assessments", function () {
```

Comment:
View archived questionnaire

Reply:
I think you meant : 

2. This test should go in administration/questionnaire/miscellaneous.test.ts

Reply:
done